editions/tw5.com/tiddlers/messages/WidgetMessage_ tm-new-tiddler.tid

```diff
@@ -36,3 +36,12 @@ To create a new tiddler with given attributes rather than from a template:
 New Tiddler
 </$button>
 ```
+
+To create a new tiddler from a template with additional attributes:
```

Comment:
Thanks @AnthonyMuscio, a couple of points:

* "template" has a special meaning in TiddlyWiki and so the docs try to avoid other usages of the word
* the term "additional attributes" might be misleading: what's going in is that we are using additional message parameters to specify field values that should be overridden

So perhaps we could say "To create a new tiddler by cloning an existing one. Parameters not starting with `$` are used to specify field values to be overridden"

Reply:
Agreed, thank you for doing it.

Reply:
In the right corner of the ui are 3 dots. You can click them, Edit the file and change it accordingly to the conversation. So there is no need to create a new PR. You can modify this one.

Reply:
Thanks @AnthonyMuscio no worries I've committed this with some tweaks in 96cb66a27ee604e551a1c7417e6c5b9e3f5884d2